<launch>


  <rosparam command='load' file='$(find dynamic_network_estimate)/param/target.yaml'/>


  <group ns='agent3'>

    <rosparam param='node_name'>
      'agent3'
    </rosparam>

    <rosparam param='initial_position'>
      [-2.0, 4.0]
    </rosparam>

    <rosparam param='initial_estimate'>
      [0.8, 1.2]
    </rosparam>

    <rosparam command='load' file='$(find dynamic_network_estimate)/param/target.yaml'/>
    <rosparam command='load' file='$(find dynamic_network_estimate)/param/estimate.yaml'/>
    <rosparam command='load' file='$(find dynamic_network_estimate)/param/simulator.yaml'/>
    <rosparam command='load' file='$(find dynamic_network_estimate)/param/planner.yaml'/>
  

    <node name='vehicle_simulator'
      pkg='dynamic_network_estimate'
      type='vehicle_simulator.py'/>

    <node name='dynamic_planner_add'
      pkg='dynamic_network_estimate'
      type='dynamic_planner_add.py'
      output='screen'/>

    <node name='estimate'
      pkg='dynamic_network_estimate'
      type='estimate.py'/>

    <node name='sensor_simulator'
      pkg='dynamic_network_estimate'
      type='sensor_simulator.py'/>

  </group>

</launch>
